Blackjack/Blackjack.py

```diff
@@ -52,7 +51,14 @@ def card_value(card):
             # If card is Ace, assign value to 11
             # todo Give player choice of using Ace as 1 or 11
             if card == 'A':
-                card_value = 11
+                print('You got an Ace! Do you choose a value of 1 or 11?')
+                ace_input = input('?> ')
```

Comment:
We probably want to accept the user input, validate that it is valid, and then assign the value. Otherwise, make the user answer again.

Reply:
Thanks for pointing this out! I fixed it all up and made it a bit cleaner as well!